### DISCUSSION TITLE
Expire and metrics does not work on version 14

### DISCUSSION THREAD

**Discussion by ArrayPlant** [wg-easy/wg-easy#1420] [Q&A]
I did everything according to `READ`.
I checked the `docker exec wg-easy printenv` further to make sure that I was not mistaken.
But nothing appeared in the web interface, and `http://ip:51821/metrics` it issues 404.

What am I doing wrong?

**Comment by anideth** [comment]
I have the same problem in 14 version, in the docker-compose config:
```
- ENABLE_PROMETHEUS_METRICS=true
- PROMETHEUS_METRICS_PASSWORD=***

``` 
When I request it I get json:
{
  "statusCode": 404,
  "statusMessage": "Cannot find static asset /metrics",
  "stack": []
}
Please help.

**Comment by anideth** [comment]
> I did everything according to `READ`. I checked the `docker exec wg-easy printenv` further to make sure that I was not mistaken. But nothing appeared in the web interface, and `http://ip:51821/metrics` it issues 404.
> 
> What am I doing wrong?

We didn't search well enough.
https://github.com/wg-easy/wg-easy/issues/1373
